test/inductor/test_aot_inductor.py

```diff
@@ -2676,6 +2676,62 @@ def forward(self, x):
         example_inputs = (torch.randn(16, 16, 16, device=self.device),)
         self.check_model(Model(), example_inputs)
 
+    def test_nested_tensor_from_jagged(self):
+        class Model(nn.Module):
+            def __init__(self):
+                super().__init__()
+                self.mlp = nn.Sequential(
+                    nn.Linear(128, 64), nn.ReLU(), nn.Linear(64, 32), nn.Sigmoid()
+                )
+
+            def forward(self, values, offsets):
+                nt = torch.nested.nested_tensor_from_jagged(values, offsets)
+                res = self.mlp(nt)
+                return res.values()
+
+        model = Model().to(device=self.device)
+
+        example_inputs_1 = (
+            torch.randn((15, 128), device=self.device),
```

Comment:
do you think it's valuable to also include a set of example inputs that have the same `values` shape but different ragged structure / offsets? e.g. two example inputs of shape `(15, 128)` with different `offsets` but same *number* of offsets